apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -13,15 +13,10 @@ export const sageAccountWebThemeOverrides: PartialDeep<ThemeOptions> = {
     MuiButton: {
       styleOverrides: {
         contained: {
-          justifyContent: 'center',
-          alignItems: 'center',
-          padding: '14px 16px',
-          height: '52px',
```

Comment:
Remove some properties from the SageAccountWeb theme that made it really hard to style these buttons consistently, especially properties that changed based on the button style (why are contained buttons taller than outlined buttons??). Everything looked ok with these gone, but if this ends up causing any major regressions, then new styles should either be applied ad-hoc or to all buttons